simple-rss-portlet/src/main/webapp/pages/view.jsp

```diff
@@ -31,35 +30,38 @@
       			<img src="${SyndFeed.image.url}" alt="${SyndFeed.image.description}" class="news-feed-img"/>
     		</a>
     	</c:if>
-		<p>${SyndFeed.description}</p>
+		<h3>${SyndFeed.description}</h3>
 	</div>
 		
 	<div class="news-items">
 		<ul>
 			<c:forEach items="${SyndFeed.entries}" var="SyndEntry" end="${maxItems}">
 				<li>
 					<c:if test="${not empty Media[SyndEntry.uri] && Media[SyndEntry.uri].image}">
-						<img src="${Media[SyndEntry.uri].url}" class="news-item-img"/>
-					</c:if>
-					<h3>
-					<c:choose>
-						<c:when test="${not empty SyndEntry.link}">
-	      					<a target="_blank" href="${SyndEntry.link}" class="news-item-title">${SyndEntry.title}</a>
-	      				</c:when>
-	      				<c:otherwise>
-		      				${SyndEntry.title}
-		      			</c:otherwise>
-		      		</c:choose>
-		      		</h3>
-	      			<span class="news-item-excerpt">${SyndEntry.description.value}</span>
-	      			
-	      			<c:if test="${not empty Media[SyndEntry.uri] && !Media[SyndEntry.uri].image}">
-						<div class="news-item-attachment">
-							<a href="${Media[SyndEntry.uri].url}">${Media[SyndEntry.uri].displayName}</a>
-							 (${Media[SyndEntry.uri].type}, ${Media[SyndEntry.uri].displayLength})
+						<div class="news-item-img">
+							<img src="${Media[SyndEntry.uri].url}" />
 						</div>
 					</c:if>
-	      				      			
+					<div class="news-item-content">
+						<h4>
+						<c:choose>
+							<c:when test="${not empty SyndEntry.link}">
+								<a target="_blank" href="${SyndEntry.link}" class="news-item-title">${SyndEntry.title}</a>
+							</c:when>
+							<c:otherwise>
+								${SyndEntry.title}
+							</c:otherwise>
+						</c:choose>
+						</h4>
```

Comment:
```suggestion
						<h2>
						<c:choose>
							<c:when test="${not empty SyndEntry.link}">
								<a target="_blank" href="${SyndEntry.link}" class="news-item-title">${SyndEntry.title}</a>
							</c:when>
							<c:otherwise>
								${SyndEntry.title}
							</c:otherwise>
						</c:choose>
						</h2>
```

Reply:
Changing these to bring it inline with other non-synoptic tools.